packages/reporting-server/migrations/models/0_20210524111029_init.sql

```diff
@@ -0,0 +1,90 @@
+-- upgrade --
+CREATE TABLE IF NOT EXISTS "authevents" (
```

Comment:
Is this generated or hand coded?

Reply:
This is generated by aerich, every time that we update a model we should create a new migration with the `aerich migrate` command and then apply it with `aerich upgrade`.

Reply:
I'll add a recommender workflow on the Readme